addons/payment/views/payment_form_templates.xml

```diff
@@ -408,37 +408,70 @@
 
     <template id="payment.no_pms_available_warning">
         <div class="alert alert-warning mt-2">
-            <strong>No suitable payment option could be found.</strong>
-            <!-- === Availability report button (for debug) === -->
-            <t t-call="payment.availability_report_button"/>
+            <strong>No payment method available for your order.</strong>
+            <!-- === Availability report button === -->
             <br/>
-            If you believe that it is an error, please contact the website administrator.
+            <t t-if="not request.env.is_admin()">
+                <p>If you believe that it is an error, please contact the website administrator.</p>
+            </t>
+            <t t-else="">
+                <t t-if="not is_payment_provider_active">
+                    <p>You didn't configure any payment provider.</p>
+                </t>
+                <t t-else="">
+                    <p>No payment method is available for
+                        <t t-esc="request.env.company.country_id.name"/>,
+                        <t t-esc="request.env.company.currency_id.name"/>.
+                    </p>
+                </t>
+            </t>
             <div class="mt-2" groups="base.group_system">
                 <a
-                    t-if="request.env.company.country_id.is_stripe_supported_country"
+                    t-if="request.env.company.country_id.is_stripe_supported_country
+                        and not is_payment_provider_active"
                     name="activate_stripe"
                     href="/web#action=payment.action_activate_stripe"
                     role="button"
                     class="btn btn-primary"
                 > ACTIVATE STRIPE </a>
                 <a
+                    t-if="request.env.user._is_system() and availability_report"
```

Comment:
No need to check _is_system as we are already in [system group](https://github.com/odoo/odoo/pull/165203/files#diff-e60fe3964a8f489ec9e2c5e892ce15bddef59a0b5eaa79808854c80aba588a0bR428). No? :thinking: